js/components/Draw.js

```diff
@@ -24,34 +27,62 @@ export default class Draw extends Component {
     this.props.navigation.navigate("Home");
   }
 
+  _changeColor = (color) => (
+    (e) => {
+      this.setState({ color })
+    }
+  )
+
   render() {
     return (
       <View style={styles.container}>
         <SketchCanvas
           ref={ref => { this.sketchCanvas = ref; }}
           style={styles.sketchCanvas}
-          strokeColor={'#F9A800'}
+          strokeColor={this.state.color}
           strokeWidth={7}
           localSourceImage={{ filename: this.props.screenProps.snapshotUri }}
         />
-
-        <Icon
-          reverse
-          name='check'
-          color='#237F52'
-          onPress={this._pressSubmit}
-        />
+        <View>
+          <Icon
+            reverse
+            color='#237F52'
+            size={18}
+            onPress={this._changeColor('#237F52')}
+          />
+          <Icon
+            reverse
+            color='#F9A800'
+            size={18}
+            onPress={this._changeColor('#F9A800')}
+          />
+          <Icon
+            reverse
+            color='#9B2423'
+            size={18}
+            onPress={this._changeColor('#9B2423')}
```

Comment:
I think it's better to give some name to those colors.

That'll prevent having different color on button and real color as well.

Reply:
true that!